include/seqan3/alignment/pairwise/edit_distance_unbanded.hpp

```diff
@@ -190,8 +218,8 @@ struct pairwise_alignment_edit_distance_unbanded
      * \param[in] _config   \copydoc config
      */
     pairwise_alignment_edit_distance_unbanded(
-        database_t _database,
-        query_t _query,
+        database_t && _database,
+        query_t && _query,
```

Comment:
I had trouble when using forwarding references, but if everything works now I'm okay with this.

Reply:
Mhmm, I mean we check the sequences to be of ViewableRange concept. So either they are views and are copied or they are l-value refs of ranges and the ref is by the deduction guide part of the type, right. So the ref can be removed here. I am always confused by this.